scripts/i18n-generator.js

```diff
@@ -0,0 +1,51 @@
+var i18nConfig = require("config").get("i18n")
+var _ = require('lodash');
+var Promise =require('bluebird');
+var fs = Promise.promisifyAll(require("fs"));
+var saw = require('string-saw')
+
+var files = ['youpin.js'];
```

Comment:
Use `const` instead of `var` if its value is not changed. (all lines above also)